nodejs/week1/homework/warmup.js

```diff
@@ -0,0 +1,21 @@
+console.log("inside warmup file succeed");
+class Circle{
+   constructor(radius){
+       this.radius = radius
+   };
+getDiameter(){
+  return this.radius*2
+};
+getCircumference(){
+    return this.radius*2*3.14159
```

Comment:
Nitpick(readability): 
Code is usually easier to read if you add spaces in between operators:
`this.radius * 2 * 3.14159`

Further, perhaps you should store the constant in a variable to explain the value:
```
const PI = 3.14159;
return this.radius * 2 * PI;
```

Lastly, `this.radius * 2` is the same as the diameter, so you could also write:
`this.getDiameter() * PI`

Reply:
wow, yes. You are totally right. nice to notice it. Smart. :)

Reply:
hI Milton. last one should be PI * this.radius * this.radius.  So we can not use this.getDiameter() * PI.  :)